NorthstarDLL/mods/modmanager.cpp

```diff
@@ -56,6 +56,14 @@ Mod::Mod(fs::path modDir, char* jsonBuf)
 
 	Name = modJson["Name"].GetString();
 
+	// Don't load blacklisted mods
+	if (!strstr(GetCommandLineA(), "-nomodblacklist") &&
```

Comment:
We should really be moving away from using `strstr` to check args, but we do it elsewhere, so I'll let this one slide.

Reply:
Ill look into this in a separate PR